config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java

```diff
@@ -245,6 +245,11 @@ public void getConfig(ProtonConfig.Summary.Cache.Compression.Builder compression
                         if (level != null) compression.level(level);
                     }
 
+                    public void getConfig(ProtonConfig.Summary.Log.Compact.Compression.Builder compression) {
+                        if (type != null)  compression.type(ProtonConfig.Summary.Log.Compact.Compression.Type.Enum.valueOf(type.name));
```

Comment:
Nitpicking: Spacing is (was already) inconsistent between the 3 getConfig() methods here.

Reply:
Unified